SokolTextGame/World.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
```

Comment:
Please find all the `using` statements that are grayed out and remove all of them. In your case here, I believe you actually don't need 99% of them. Please only leave the ones that are in full color and remove those that are grayed out. This means not just this particular using statement and not even this particular file, but all the source files of the game. Look carefully and remove all `using`s that are unnecessary.